src/config/constants/types.ts

```diff
@@ -1,4 +1,5 @@
 import BigNumber from 'bignumber.js'
+import { Token as SDKToken } from '@pancakeswap/sdk'
```

Comment:
There is another `Token` type defined below and it is not the same as `Token` in SDK (in SDK symbol is optional)